Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.8.10
+RUN apt-get update -y
+RUN apt-get upgrade -y
+RUN apt-get install libfreetype6-dev libxft-dev fortune -y
```

Comment:
What are libfreetype and libxft used for? I was able to build & run okay without them.

Reply:
kinda suss ngl

stay safe king!

Reply:
Just some font libs that @olliequ installed in his version for font compatibility with Matplotlib. If you want, I can remove them.

Reply:
They were recommended to be installed on my DO Ubuntu server by the guide I was using: 
> For some reason, Matplotlib and a few other packages have issues with Ubuntu, but you can solve these by running:
> sudo apt-get install libfreetype6-dev libxft-dev
> sudo apt-get install python-setuptools

The guide: https://towardsdatascience.com/lessons-learned-digital-ocean-for-python-3-e2442db4246f